test/Types/Interval.purs

```diff
@@ -36,8 +36,13 @@ import Types.Interval
 suite :: TestPlanM (EraSummaries -> SystemStart -> Effect Unit) Unit
 suite = do
   group "Interval type" do
-    test "Inverse posixTimeToSlot >>> slotToPosixTime " $ testPosixTimeToSlot
-    test "Inverse slotToPosixTime >>> posixTimeToSlot " $ testSlotToPosixTime
+    -- These tests depend on assumptions about testnet history.
+    -- We disabled them during transition from `testnet` to `preprod` networks.
+    -- https://github.com/Plutonomicon/cardano-transaction-lib/issues/945
+    skip $ test "Inverse posixTimeToSlot >>> slotToPosixTime " $
+      testPosixTimeToSlot
+    skip $ test "Inverse slotToPosixTime >>> posixTimeToSlot " $
+      testSlotToPosixTime
```

Comment:
Ideally we disable these tests in `Test.Integration` and not here, as `Test.Unit` is able to pass the tests fine using the fixtures below